app/views/merchants/show.html.erb

```diff
@@ -0,0 +1 @@
+<%= link_to 'View all items for sale', merchant_items_index_path %>
```

Comment:
May need to update the path here if we change to a merchant namespace + resourceful route